packages/survey-creator-core/src/components/tabs/theme-custom-questions/element-settings.ts

```diff
@@ -21,23 +22,17 @@ function getElementsJSON() {
     },
     {
       type: "spinedit",
-      name: "corner",
```

Comment:
I don't like `"colorsettings"` above. Can we rename it to simply `"color"`?

Reply:
The color editor already exists. This editor is a combined color and transparency editor. We need to come up with something else.